pkg/api/aim/runs.go

```diff
@@ -240,6 +240,7 @@ func GetRunsActive(c *fiber.Ctx) error {
 	log.Debugf("getRunsActive namespace: %s", ns.Code)
 
 	q := struct {
+		Limit          int  `query:"limit"`
```

Comment:
This is not supported by the UI at this stage (and probably never will)

Reply:
correct. I though that we want to just limit default case.

Reply:
for now I would suggest to simply hard-code the limit to 100.

Reply:
ok.

Reply:
hardcoded.